whois-endtoend/src/test/groovy/net/ripe/db/whois/spec/update/NonAllocationAttributeValidationSpec.groovy

```diff
@@ -81,9 +81,10 @@ class NonAllocationAttributeValidationSpec extends BaseQueryUpdateSpec {
         ack.summary.nrFound == 1
         ack.summary.assertSuccess(0, 0, 0, 0, 0)
         ack.summary.assertErrors(1, 0, 1, 0)
-        ack.countErrorWarnInfo(1, 0, 0)
+        ack.countErrorWarnInfo(2, 0, 0)
         ack.errors.any { it.operation == "Modify" && it.key == "[inetnum] 192.168.1.0 - 192.168.1.255" }
         ack.errorMessagesFor("Modify", "[inetnum] 192.168.1.0 - 192.168.1.255") == [
+                "\"mnt-lower:\" attribute not allowed for resources with \"ASSIGNED PI:\" status",
```

Comment:
Where did this error come from? Can you remove mnt-lower from test data everywhere (unless you're testing for it specifically)

Reply:
Remove "mnt-lower" from "modify inetnum, change sponsoring-org by lir mntner is not possible" test

Reply:
Done